app/views/effective/alerts/_alert.html.haml

```diff
@@ -0,0 +1,4 @@
+%div{class: "alert alert-warning", role: 'alert'}
+  .container
+    .row
+      .col= alert.body
```

Comment:
I think this could just be a div. I don't think we need the container, row or col